Bottom: a41df6208675d43dfa794612370e519219a8d002
Top:    167fd9d1ec74c683f56e49b89f3c793f892857f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:50 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
